backend/routers/store_menus.go

```
@@ -22,6 +26,102 @@ func (s *Server) GetMenusByStoreID(c *gin.Context, storeID uuid.UUID) {
+		})
+		return
+	}
+
+	image, err := form.Image.Open()
+	if err != nil {
+		c.JSON(http.StatusBadRequest, ErrorResponse{
+			Message: "画像を読み込めませんでした。",
+		})
+		return
+	}
+	defer image.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `image.Close` is not checked

(errcheck)